### PR TITLE
Updated to version v2.0.5

### PR DESCRIPTION
**Pull request by WillAWS**
*Issue #, if available:*

*Description of changes:*

- Update parameter names for consistency
- Refactor to reduce code complexity
- Update client configs to latest sdk version
- Fix broken URLs in README

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tbelmega — APPROVED** [review]

File: deployment/aws-fms-automations.template [hunk 1]
```diff
@@ -326,27 +327,11 @@
...
-                      "Solution",
-                      "SolutionVersion"
-                    ]
-                  }
-                ]
+            "USER_AGENT_PREFIX": {
+              "Fn::FindInMap": [
+                "CommonResourceStackMap",
+                "Solution",
+                "UserAgentPrefix"
```
Comment (gsingh04) [inline deployment/aws-fms-automations.template:334]: looks like wrong user agent prefix? shouldn't it follow the standard `AwsSolution/<SolutionId>/<SolutionVersion>`

File: source/services/policyManager/lib/exports.ts [hunk 2]
```diff
@@ -4,8 +4,11 @@
 /**
  *
  */
+import { UserAgent, UserAgentPair } from "@aws-sdk/types";
+
+const userAgentPair : UserAgentPair = [`${process.env.USER_AGENT_PREFIX}/${process.env.SOLUTION_ID}`, `${process.env.SOLUTION_VERSION}`]
+export const customUserAgent : UserAgent = [userAgentPair];
```
Comment (WillAWS) [inline source/services/policyManager/lib/exports.ts:10]: @gsingh04 the useragent prefix is only parit of the user agent string built as a userAgentPair.

Comment (gsingh04) [reply source/services/policyManager/lib/exports.ts:10]: do we have these env variables configured on the lambda resource?

Comment (WillAWS) [reply source/services/policyManager/lib/exports.ts:10]: yes

**Pull request merged**